builder/alicloud/ecs/artifact.go

```diff
@@ -136,7 +136,7 @@ func (a *Artifact) Destroy() error {
 		if len(errors) == 1 {
 			return errors[0]
 		} else {
-			return &packer.MultiError{Errors: errors}
+			return &packersdk.MultiError{Errors: errors}
```

Comment:
This comment is just out of curiosity in case you have the context. Is there a reason why Packer doesn't use the [go-multierror](https://github.com/hashicorp/go-multierror) pkg? 

It's possible that Packer's multierror is the predecessor to the go-multierror pkg, but I've never looked.

Reply:
I've never looked, either. At this point, it's just convention. We could definitely look at swapping it out though; one less thing to maintain.